packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -369,6 +381,12 @@ export default {
                 this.$emit(EventNames.CheckoutSetupGuestSuccess);
             } catch (thrownErrors) {
                 this.$emit(EventNames.CheckoutSetupGuestFailure, thrownErrors);
+
+                this.$logger.logError(
```

Comment:
Out of interest – what happens if `this.$logger` doesn't exist? Do these calls just silently fail?

Reply:
😨 I'm not sure actually. Do I need to have something in somewhere to check it's there when mounted?